.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,38 @@
+name: Release Drafter
+
+on:
+  schedule:
+    - cron: "*/5 * * * *"
+  pull_request_target:
+    branches:
+      - "**"
+    paths-ignore: ["docs/**", "**.md"]
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+  update_release_draft:
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Current Branch (Fast)
+        uses: actions/checkout@v4
+
+      - id: create_release
+        uses: release-drafter/release-drafter@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set Output for Later Jobs
+        id: set_output
+        run: |
+          echo "create_release_name=${{ steps.create_release.outputs.name }}" >> $GITHUB_OUTPUT
+          echo "create_release_id=${{ steps.create_release.outputs.id }}" >> $GITHUB_OUTPUT
+    outputs:
+      create_release_name: ${{ steps.set_output.outputs.create_release_name }}
+      create_release_id: ${{ steps.set_output.outputs.create_release_id }}
```

Comment:
this output will not get reflected in the tailcall build. it will introduce a bug